Packs/CyberArkEventCollector/Integrations/CyberArkEventCollector/README.md

```diff
@@ -0,0 +1,54 @@
+CyberArk logs event collector integration for Cortex XSIAM.
+This integration was integrated and tested with version 22.4 of CyberArk Event Collector.
+
+## Configure CyberArk Event Collector on Cortex XSIAM
+
+1. Navigate to **Settings** > **Configurations** > **Data Collection** > **Automation & Feed Integrations**.
+2. Search for CyberArk Event Collector.
+3. Click **Add instance** to create and configure a new integration instance.
+
+    | **Parameter** | **Description** | **Required** |
+    | --- | --- | --- |
+    | Server URL |  | True |
+    | App ID |  | True |
+    | User name | The user name (e.g.`admin@example.com`). | True |
+    | Password |  | True |
+    | First fetch time (&lt;number&gt; &lt;time unit&gt;, e.g., 12 hours, 1 day, 3 months). Default is 3 days. |  | True |
+    | The maximum number of events per fetch. Default is 100. |  | True |
```

Comment:
Why is it mandatory if it has a default, is it required by the api ?

Reply:
Because the user can remove it (there are no default value inside the integration code)